ee/clickhouse/models/test/__snapshots__/test_property.ambr

```diff
@@ -147,7 +147,7 @@
       ))
     ''',
     dict({
-      'global_cohort_id_0': 47,
+      'global_cohort_id_0': 1,
```

Comment:
Huh, these snapshots keep getting changed.

Reply:
hmmmm I don't know much about these ambr tests, maybe ask in dev